frontend/src/components/shared/mainInfo/MainInfo.tsx

```diff
@@ -25,7 +25,12 @@ const MainInfo: React.FC = () => {
         <div className="sex-wrapper">
           <SelectWithFormik label="Sex" name="sex" options={sex} />
         </div>
-        <InputWithFormik label="Birthday" name="birthday" type="date" maxAge={new Date().toISOString().substring(0, 10)} />
+        <InputWithFormik
+          label="Birthday"
+          name="birthday"
+          type="date"
+          maxAge={new Date().toISOString().substring(0, 10)}
```

Comment:
Please use dayJs to work with date
please rename field to reflect its purpose - maxAge contains current Date as string?

Reply:
It will be fixed by @romanazaruk